content/reduced-form-specification.md

```diff
@@ -178,8 +181,7 @@ weather data that has been transformed at the grid level.
 
 ### Aggregation-before-transformation
 
-% Let us try to understand these
```

Comment:
was this supposed to be commented out?

Reply:
Those lines should have just been deleted. I've don that now.